package.json

```diff
@@ -4,20 +4,21 @@
   "license": "MIT",
   "scripts": {
     "build": "npm run clean && mkdirp build && webpack --progress --colors",
+    "build:esmodules": "$(npm bin)/tsc --project ./tsconfig.json --module esnext",
```

Comment:
I don't think you need `$(npm bin)/`. Isn't it automatically added to the `$PATH` for all `scripts` in `package.json`? E.g., we call `standard-changelog` directly below. The `$(...)` syntax is not Windows-compatible.

Reply:
ah ya you're right - don't need it in this context, and it is not windows compat